cirq-aqt/cirq_aqt/aqt_device.py

```diff
@@ -218,13 +223,148 @@ def simulate_samples(self, repetitions: int) -> cirq.Result:
         return result
 
 
-class AQTTargetGateset(cirq.ion.ion_device._IonTargetGateset):
-    pass
-
+class AQTTargetGateset(transformers.TwoQubitCompilationTargetGateset):
```

Comment:
Can we add a docstring?

Reply:
> Can we add a docstring?

@tanujkhattar - could you please help with this?  (I think you are the original author of that class)

Reply:
added a suggestion, PTAL.

Reply:
> added a suggestion, PTAL.

committed, thank you!